Packs/MISP/Integrations/MISPV3/MISPV3.yml

```diff
@@ -2486,11 +2486,31 @@ script:
     - contextPath: MISP.Warninglist.Lists
       description: Name of warninglists where the value was found.
       type: string
-  dockerimage: demisto/pymisp2:1.0.0.62395
-  feed: false
-  isfetch: false
-  longRunning: false
-  longRunningPort: false
+  - arguments:
+    - description: The email address of the new user to be added.
+      name: email
+      required: true
+      type: textArea
+    - description: ID number indicating which organization the new user will be added to.
+      name: org_id
+      required: true
+      type: textArea
+    - description: Role of the new user to be added
+      name: role_id
+      required: true
+      type: textArea
+    - description: A password for the new user.
```

Comment:
Please add to the description the password requirements (length, numbers or letters etc.).